pkgs/development/python-modules/adafruit-nrfutil/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pyserial
+, click
+, ecdsa
+, behave
+}:
+
+buildPythonPackage rec {
+  pname = "adafruit-nrfutil";
+  version = "0.5.3.post16";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "b72103dc8e50f92951f512fc2b6f2c621dac1614531ddf8c0be3b13d020e24ad";
+  };
+
+  propagatedBuildInputs = [
+    pyserial
+    click
+    ecdsa
+    behave
+  ];
+
+  # package does not include tests
+  doCheck = false;
+
```

An additional safe-guard.

```suggestion

  pythonImportsCheck = [
    "nordicsemi"
  ];

```

---

I am quite new to Nix packaging - how does this change help? The unit tests should eventually cover the testing (?)

---

> The unit tests should eventually cover the testing (?)

Unit tests usually import the module but if the tests are not executed then `pythonImportsCheck` helps to ensure that the module can be imported.

It is a cheap check. It happens quite often that the tests have to be disabled due to whatever reason during the life-time of a package.

---

Thank you for the explanation !